common/src/main/java/com/gregtechceu/gtceu/common/data/GTMachines.java

```diff
@@ -1011,8 +1013,7 @@ public static BiConsumer<ItemStack, List<Component>> createTankTooltips(String n
                     tooltip.add(Component.translatable("gtceu.machine.cleanroom.tooltip.7"));
                     //tooltip.add(Component.translatable("gtceu.machine.cleanroom.tooltip.8"));
                     if (LDLib.isModLoaded(GTValues.MODID_APPENG)) {
-                        // TODO ae2 compat
-                        //tooltip.add(Component.translatable(AEConfig.instance().isFeatureEnabled(AEFeature.CHANNELS) ? "gtceu.machine.cleanroom.tooltip.ae2.channels" : "gtceu.machine.cleanroom.tooltip.ae2.no_channels"));
+                        tooltip.add(Component.translatable(AEConfig.instance().getChannelMode() == ChannelMode.INFINITE ? "gtceu.machine.cleanroom.tooltip.ae2.no_channels" : "gtceu.machine.cleanroom.tooltip.ae2.channels"));
```

Comment:
This is problematic because that class is outside of the public API, and the channel mode is not synced to the client.

Reply:
and how could i fix that? is there a method to use?

Reply:
Not atm, best to leave it out for now. You'd have to sync it yourself even if it existed...

Reply:
fun fact: i believe those tooltips are synced from server

Reply:
but sure, i'll remove that

Reply:
Ah if they're loaded from server it might be reasonable for us to add an API - but the value can change at any time so it's not great.